docs/1_Getting-Started/1_getting-started.md

```diff
@@ -1,5 +1,9 @@
 # Getting Started
 
+## Docker
+
+Docker is the recommended way to use the LUCI ROS2 SDK. Instructions for how to get started with the docker container can be found in [Getting started with Docker](./2_docker.md)
+
```

Comment:
Full path no relatives, it breaks when the site is made. Look at the setup-bash link for example

Reply:
After talking with @luciguci, this one should be okay because it is in the same directory